packages/binding.core/src/event.ts

```diff
@@ -42,8 +46,9 @@ export const eventHandler = {
             handlerReturnValue = handler.apply(possiblyUpdatedViewModel, argsForHandler)
           }
         } finally {
-          if (handlerReturnValue !== true) {
-            // Normally we want to prevent default action. Developer can override this be explicitly returning true.
+          if (handlerReturnValue !== true && (preventDefault === undefined || unwrap(preventDefault))) {
```

Comment:
I find this logic hard to grok (it always has been).

Perhaps we could break out the `preventDefault === undefined || unwrap(preventDefault)` into a separate, descriptive variable.

I'm also ok with another variable name that _may_ be clearer e.g. `requireDefault` or `doNotPreventDefault` (and whose truthiness matches an `undefined` state, making it a bit easier to comprehend)

Reply:
Another thought I had was to ignore the return value completely when preventDefault is specified in the handler. That might be clearer and also less surprising when a in inline lambda handler accidentally returns a Boolean value.

Reply:
I made it so preventDefault in the descriptor takes precedence over the return value. Lmk what you think of that -- I think this will be the least surprising behavior while maintaining backwards compatibility.

Reply:
Yeah, I think the most expressive and intuitive is to have the explicit argument determine the behaviour, and override any implied return value. Ie. if someone says preventDefault is X who are we to argue? 😀